packages/mds-db/index.ts

```diff
@@ -1475,6 +1476,22 @@ async function writePolicy(policy: Policy) {
   })
 }
 
+async function readRule(rule_id: UUID): Promise<Rule> {
+  const client = await getReadOnlyClient()
+  const sql = `SELECT * from ${schema.POLICIES_TABLE} where EXISTS(SELECT 1 FROM json_array_elements(${schema.POLICIES_COLS[1]}->'rules') elem WHERE (elem->'rule_id')::jsonb ? '${rule_id}');`
+  const res = await client.query(sql).catch(err => {
+    throw err
+  })
+  if (res.rowCount !== 1) {
+    throw new Error(`invalid rule_id ${rule_id}`)
+  } else {
+    const policy = res.rows[0].policy_json as Policy
```

Comment:
I like to use destructuring rather than `rows[0]` to get the row. `const [row] = res.rows` or even `const [{policy_json}] = res.rows`

Reply:
ooh nice